scripts/launch_slack_bot.py

```diff
@@ -195,29 +194,16 @@ def get_message_chunks(self) -> List[str]:
         # to launch each command in self._get_commands() sequentially, and
         # store all the stdout/stderr into output.txt.
         user = self._user
-        dir_name = SUPERCLOUD_USER_TO_DIR[user]
-        assert dir_name.endswith("predicators/")
-        dir_name_up = dir_name[:-12]
         # 1) Always start by clearing the current output.txt.
         commands = ["rm -f output.txt"]
         for command in self._get_commands():
             # 2) SSH into the user's login server.
             commands.append(
-                f"ssh {user}@{SUPERCLOUD_LOGIN_SERVER} "
+                f"ssh {user}@{SUPERCLOUD_LOGIN_SERVER} -t 'bash -ic \""
                 # 3) Change into the desired directory and source
-                # the user's conda environment.
-                f"'cd {dir_name} && source {CONDA} predicators "
-                # 4) This step is tricky! For some reason, when you
-                # SSH, the PYTHONPATH isn't preserved, so we update
-                # it here to be the directory one level up from the
-                # predicators repository, so that the imports in our
-                # repository work as expected (from predicators...).
-                f"&& export PYTHONPATH=$PYTHONPATH:{dir_name_up} "
-                # 5) Similarly, update the PYTHONHASHSEED.
-                "&& export PYTHONHASHSEED=0 "
-                # 6) Run the actual command, appending both stdout
-                # and stderr onto the end of output.txt.
-                f"&& {command}' >> output.txt 2>&1")
+                # the user's conda environment. Then, run the actual command,
+                # appending both stdout and stderr onto the end of output.txt.
+                f"predicate && {command}\"' >> output.txt 2>&1")
```

Comment:
Looks good but what is this `predicate` command?

Reply:
Thanks for reviewing. It's a bash alias that switches directories and activates the right conda env. Since this bot only gets run on my account it should be okay